crates/agglayer-certificate-orchestrator/src/certificate_task.rs

```diff
@@ -314,6 +302,14 @@ where
         self.header.status = status;
         Ok(())
     }
+
+    async fn send_to_network_task(
+        &self,
+        msg: NetworkTaskMessage,
+    ) -> Result<(), CertificateStatusError> {
+        trace!(?msg, "Sending message to network task");
```

Comment:
Nit: call the log key "message" rather than "msg"
```suggestion
        trace!(message = ?msg, "Sending message to network task");
```

(or rename the varible)